sdk/openai/azure-ai-openai-assistants/src/test/java/com/azure/ai/openai/assistants/AzureFileSearchAsyncTest.java

```diff
@@ -122,4 +127,37 @@ public void basicFileSearch(HttpClient httpClient, AssistantsServiceVersion serv
                 .verifyComplete();
         });
     }
+
+    @Disabled("file_search tools are not supported in Azure")
```

Comment:
It's a bit confusing for me, this test is asserting an error, but at the same time is disabled. Should it be enabled?

Reply:
Yeah. It is working now. Let me remove it.